### PR TITLE
Svelte 5 evlim

### PR DESCRIPTION
**Pull request by scosman**
## Release Notes

- **New Features**
	- Introduced Handlebars for email template rendering, enhancing customization options for welcome emails.
	- Added new `welcome_email_text.hbs` template for user welcome messages.
	- Improved error handling and validation in account management features.

- **Bug Fixes**
	- Enhanced validation for email and password updates to ensure proper user input.

- **Documentation**
	- Updated email setup instructions for clarity, particularly regarding Handlebars syntax and file formats.

- **Refactor**
	- Transitioned to using `$state()` for state management in various components, improving reactivity.
	- Updated component props handling to utilize TypeScript interfaces for better type safety.

- **Chores**
	- Updated package dependencies to their latest versions for improved performance and security.
	- Removed deprecated email component and updated related configurations.

### CODE REVIEW

**Comment by scosman** [comment]
@evlim any feedback? I think it's ready?

File: src/routes/(admin)/account/(menu)/+layout.svelte [hunk 1]
```diff
@@ -68,7 +73,7 @@
         <a
           href="/account"
           class={adminSection === "home" ? "active" : ""}
-          on:click={closeDrawer}
+          onclick={closeDrawer}
```
[the remark this answers is not in the document]
Comment (scosman) [reply src/routes/(admin)/account/(menu)/+layout.svelte:76]: that's incorrect. Svelte 5 moved back to "onclick". This project is svelte 5.

**Comment by evlim** [comment]
@scosman Should be good after #165

**Comment by evlim** [comment]
@scosman Looks like there's nothing else with `svelte/legacy`

**Pull request merged**